source/globalCommands.py

```diff
@@ -124,6 +124,30 @@
 NO_SETTINGS_MSG = _("No settings")
 
 
+def toggleBooleanValue(
+	configSection: str,
+	configKey: str,
+	# Translators: The message announced when toggling the configKey setting.
+	enabledMsg: str,
+	# Translators: The message announced when toggling the configKey setting.
+	disabledMsg: str,
+) -> str:
+	"""
+	Toggles a boolean value in the configuration and returns the appropriate message.
+
+	:param configSection: The configuration section containing the boolean key.
+	:param configKey: The configuration key associated with the boolean value.
+	:param enabledMsg: The message for the enabled state.
+	:param disabledMsg: The message for the disabled state.
+	:return: The translatable message to be displayed to the user.
+	"""
+	currentValue = config.conf[configSection][configKey]
+	newValue = not currentValue
+	config.conf[configSection][configKey] = newValue
+
+	return _(enabledMsg) if newValue else _(disabledMsg)
```

Comment:
This is not valid. I also thing ui.message should be here, and this should return none.

```suggestion
	msg = enabledMsg if newValue else disabledMsg
	ui.message(msg)
```